### DISCUSSION TITLE
Self-Managed Pro Pricing

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8778] [Help]
Hi there,

Please correct me if I'm wrong, but if understand correctly, Open source is free forever, and there are no limitations other that a maximum of 10 plugins and user groups.

If I want to get the user groups functionality, I've got to upgrade to one of the paid plans. These also mention a number of day passes per month. Now I'm wondering whether the no off day passes start to apply in the self managed version.

Thanks in advance and best regards, Dick

**Comment by joebudi** [comment]
Hey! Yes, your understanding is right. Our open source plan is not limited. When you upgrade, you get additional features, and you are then charged based on day passes. We're always open to feedback on pricing - we only launched it last month. If you like, I'd be happy to jump on a call to discuss more.

**Reply by ecxs** [reply]
Hi @joebudi I would really like to discuss this with you. I'm coming from an entirely different background, and have been using proprietary software for the last 40 years. So open source, fair code, etc. license is fairly new to me. 

I appreciate that we all have to make a living somehow, and I have no problem whatsoever to pay for added value. However, before I decided to start using MySQL, I had been playing around with Supabase, and I really liked it … until I started self hosting, when I found out that the self hosted version was severely crippled.

I hope you appreciate that it's important for me to see where this is heading, before I seriously invest time in BudiBase.

Again, I truly appreciate that everybody has to pay the bills, including myself :wink:

**Reply by joebudi** [reply]
Send me an email ( joe at budibase dot com ), and we'll go from there.
